src/connection/passwordless/actions.js

```diff
@@ -17,19 +17,7 @@ import {
 import { phoneNumberWithDiallingCode } from '../../field/phone_number';
 import * as i18n from '../../i18n';
 
-export function requestPasswordlessEmail(id) {
-  validateAndSubmit(id, ['email'], m => {
-    sendEmail(m, requestPasswordlessEmailSuccess, requestPasswordlessEmailError);
-  });
-}
-
-export function requestPasswordlessEmailSuccess(id) {
-  swap(updateEntity, 'lock', id, lock => {
-    return setPasswordlessStarted(l.setSubmitting(lock, false), true);
-  });
-}
-
-function startPasswordlessErrorMessage(m, error, medium) {
+function getErrorMessage(m, error) {
```

Comment:
what's `m` for? it's not used

Reply:
m, across the lock codebase, is the global state object of lock